windows/win_msi.py

```diff
@@ -45,6 +45,16 @@
         description:
             - Path to a file created by installing the MSI to prevent from
               attempting to reinstall the package on every run
+    wait:
+        version_added: "2.0"
+        description:
+            - Specify whether to wait for install or uninstall to complete before continuing.
+        choices:
```

Comment:
Why so many choices that mean the same thing?  I know ConvertTo-Bool supports them all, but I feel a module shouldn't have more than one option that means the same thing.